src/main/java/seedu/address/model/person/insurance/InsurancePlansManager.java

```diff
@@ -16,14 +19,18 @@ public class InsurancePlansManager {
             + "has already been added to this client: %2$s";
     public static final String PLAN_NOT_DETECTED_MESSAGE = "This plan with id: %1$s, "
             + "has not been added to this client: %2$s";
+    public static final String DUPLICATE_CLAIM_ID_MESSAGE = "This claim with id: %1$s "
+            + "has already been added to this client: %2$s";
 
     private final ArrayList<InsurancePlan> insurancePlans;
+    private final HashSet<String> claimIds;
 
     /**
      * Constructs an empty InsurancePlans list.
      */
     public InsurancePlansManager() {
         this.insurancePlans = new ArrayList<>();
```

Comment:
Should this be new ArrayList<InsurancePlan>() instead? that was at least how we were taught in CS2030 😅